core/web/router.go

```diff
@@ -306,45 +306,70 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 	userOrEI.GET("/ping", ping.Show)
 }
 
-func guiAssetRoutes(box packr.Box, engine *gin.Engine) {
-	boxList := box.List()
+// This is higher because it serves main.js and any static images. There are
+// 5 assets which must be served, so this allows for 20 requests/min
+var staticAssetsRateLimit = int64(100)
+var staticAssetsRateLimitPeriod = 1 * time.Minute
+var indexRateLimit = int64(20)
+var indexRateLimitPeriod = 1 * time.Minute
```

Comment:
Is it worth making these configurable as ENV vars?

Reply:
We discussed it, and decided that we didn't want to add too many additional env vars, however I'm open to changing it

@spooktheducks What do you think?